openreview/venue_request/process/revisionProcess.py

```diff
@@ -28,6 +28,9 @@ def process(client, note, invitation):
                 if matching_invitation:
                     matching_invitation.cdate = openreview.tools.datetime_millis(submission_deadline)
                     client.post_invitation(matching_invitation)
+                revision_invitation = openreview.tools.get_invitation(client, conference.get_invitation_id('Revision'))
+                if revision_invitation:
+                    conference.setup_post_submission_stage(hide_fields=forum_note.content.get('hide_fields', []))
```

Comment:
this means we are going to call this every time there is a revision invitation and sometimes there is a revision invitation after the deadline. The function `setup_post_submission_stage` is expensive so I suggest the following:

- Check if now is between first and second deadline
- Check if the Revision invitation exists.
- Update the Revision invitation duedate and expdate with the new second deadline value.